saleor/core/utils/operators.py

```diff
@@ -0,0 +1,6 @@
+from typing import Any, Optional
+
+
+def xor_none(a: Optional[Any], b: Optional[Any]) -> bool:
```

Comment:
I think this would be easier to understand if it was named `exactly_one` and if it accepted an iterable, like `any` and `all`?